pkg/restapi/oidc/operations_test.go

```diff
@@ -7,10 +7,12 @@ SPDX-License-Identifier: Apache-2.0
 package oidc
 
 import (
-	"context"
 	"crypto/rand"
 	"errors"
 	"fmt"
+	"github.com/trustbloc/edge-agent/pkg/restapi/common/store/cookie"
+	"github.com/trustbloc/edge-agent/pkg/restapi/common/store/tokens"
+	"github.com/trustbloc/edge-agent/pkg/restapi/common/store/user"
```

Comment:
wrong import location, need to move down along with oidc2 at the bottom of imports

Reply:
linter used to fail for these errors